packages/imageeditor/src/index.js

```diff
@@ -129,8 +130,10 @@ class ImageEditor extends PureComponent {
     }
 
     handleCanvasImageChange = () => {
-        const newCropRect = this.canvasRef.getCroppingRect();
-        this.props.onCropChange(newCropRect);
+        if (!this.props.readOnly) {
```

Comment:
Do you want to cover the line in the test as well?

Reply:
yeah it should be covered